cmd/agent/flow_run.go

```diff
@@ -108,11 +109,15 @@ func (fr *flowRun) Run(configFile string) error {
 	reload := func() error {
 		flowCfg, err := loadFlowFile(configFile)
 		if err != nil {
+			instrumentation.ConfigMetrics.InstrumentLoad(true)
```

Comment:
Could this be simplified to one `defer instrumentation.ConfigMetrics.InstrumentLoad(err != nil)`?